docs/dev/libvirt-howto.md

```diff
@@ -52,6 +52,23 @@ polkit.addRule(function(action, subject) {
 EOF
 ```
 
+### Enable IP Forwarding
+
+Libvirt creates a bridged connection to the host machine, but in order for the
+network bridge to work IP forwarding needs to be enabled. To enable it for the
+current boot:
+
+```
+sudo sysctl net.ipv4.ip_forward=1
+```
+
+or to persist the setting across reboots (recommended):
+
+```
+echo "net.ipv4.ip_forward = 1" | sudo tee /etc/sysctl.d/99-ipforward.conf
+sudo sysctl -p /etc/sysctl.d/99-ipforward.conf
```

Comment:
Maybe mention the query `sysctl net.ipv4.ip_forward` earlier in this section?  We don't want folks worrying about setting the property, especially via `/etc`, if they already have forwarding enabled.